Test failure in ydb/tests/compatibility/test_statistics.py.TestAnalyzeRollingUpdate.test[rolling_stable-26-1-1_to_prestable-26-2]: planner selectivity estimate is 0

## Summary

During rolling upgrade from `stable-26-1-1` to `prestable-26-2`, the planner selectivity estimate for a column-store table returns 0 instead of a value reflecting actual row count. The test expects `estimate >= expected_count * 0.5` (i.e., >= 50 for 100 rows), but gets 0.

The test is flaky — it failed in try 1 but passed in try 2 and try 3, suggesting a timing/propagation issue with ANALYZE statistics in a mixed-version cluster.

## Affected test (flaky, failed in try 1 only)

- `ydb/tests/compatibility/test_statistics.py.TestAnalyzeRollingUpdate.test[rolling_stable-26-1-1_to_prestable-26-2]`

## Build Information

- **Commit:** `7a77c4d3eab36d99c9b938656150b779a90e1e10`
- **Build Preset:** relwithdebinfo
- **Branch:** main
- **Date:** 2026-02-26

## Error details

```
ydb/tests/compatibility/test_statistics.py:350: in test
    assert estimate >= expected_count * 0.5
E   assert 0 >= (100 * 0.5)
```

Context from log — the EXPLAIN output for `SELECT count(*) FROM cs_table WHERE int_val < 10` shows `E-Rows: 0` for the TableFullScan operator, indicating the planner has no statistics available for this table.

## Test Reports

- **Workflow run:** [Regression-run_compatibility #448](https://github.com/ydb-platform/ydb/actions/runs/22429562196)
- **HTML reports:** [Try 1](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/22429562196/ya-main-x86-64/try_1/ya-test.html) | [Try 2](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/22429562196/ya-main-x86-64/try_2/ya-test.html) | [Try 3](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/22429562196/ya-main-x86-64/try_3/ya-test.html)

**Logs:**
- [try 1](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/22429562196/ya-main-x86-64/try_1/artifacts/logs/ydb/tests/compatibility/test-results/py3test/testing_out_stuff/test_statistics.py.TestAnalyzeRollingUpdate.test.rolling_stable-26-1-1_to_prestable-26-2.log)
- [Artifacts Index](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/22429562196/ya-main-x86-64/index.html)

## Diagnosis

During rolling upgrade from `stable-26-1-1` to `prestable-26-2`, the ANALYZE statistics for a column-store table are not properly propagated or available, resulting in a zero selectivity estimate. The test is flaky — it passed on subsequent retries, suggesting a timing issue where statistics need more time to propagate in a mixed-version cluster.

--- a/ydb/tests/compatibility/test_statistics.py
+++ b/ydb/tests/compatibility/test_statistics.py
@@ -5,6 +5,8 @@ import pytest
 import random
 import threading
 
+import requests
+
 from ydb.tests.library.harness.util import LogLevels
 from ydb.tests.library.compatibility.fixtures import RestartToAnotherVersionFixture, RollingUpgradeAndDowngradeFixture
 from ydb.tests.library.common.wait_for import wait_for
@@ -328,6 +330,28 @@ class TestAnalyzeRollingUpdate(RollingUpgradeAndDowngradeFixture):
         ROW_COUNT = 1000
         self.create_tables()
         self.write_data(ROW_COUNT)
+
+        def base_stats_ready(table_name):
+            mon_port = list(self.cluster.slots.values())[0].mon_port
+            try:
+                response = requests.get(
+                    f"http://localhost:{mon_port}/actors/statservice",
+                    params={
+                        "action": "probe_base_stats",
+                        "path": f"{self.database_path}/{table_name}",
+                        "json": 1,
+                    })
+            except requests.exceptions.RequestException:
+                return False
+
+            logger.debug(f"{table_name} table base stats: {response.json()}")
+            if response.status_code == 200:
+                return response.json()["row_count"] > 0
+            return False
+
+        assert wait_for(
+            lambda: all(base_stats_ready(t) for t in self.tables), timeout_seconds=150), \
+            "base stats not ready"
 
         def try_analyze(session_pool, table_name):
             try:

